setup.py

```diff
@@ -103,7 +103,7 @@ def run_tests(self):
     name=PACKAGE,
     version=__version__,
     description=__desc__,
-    long_description=open("{}/README.md".format(DIR)).read(),
+    long_description=open("{}/README.md".format(DIR), encoding = "utf-8").read(),
```

Comment:
Add `import io` at the top and change this line to the below to prevent breaking Python 2.7.

    long_description=io.open("{}/README.md".format(DIR),
                             encoding='utf-8').read(),

Reply:
Done